src/ui/popup.tsx

```diff
@@ -160,21 +172,43 @@ class Popup extends React.Component<IProps, IState> {
                   src={sentryLogo}
                 />
                 <ul>
+                  <li>
+                    <span className="location">
+                      {this.state.sentryErrorSampleRate ? (
+                        <span className={(this.state.sentryErrorSampleRate < ACCEPTABLE_SAMPLE_RATE) ? "warning" : "success"}>
+                          Sampling Errors at {this.state.sentryErrorSampleRate}%
+                        </span>
+                      ) : (
+                        <span className="warning">
+                          Could not detect an error sample rate
+                        </span>
+                      )
+                    }
+                    </span>
+                  </li>
                   <li>
                     {this.state.usesSentryPerformance ? (  
-                      <p className="location success">
-                        Uses Sentry performance!
-                      </p>
+                      <span className="location">
+                        <span> Using Sentry performance</span> <br/>
+                        <span className={(this.state.sentryPerformanceSampleRate < ACCEPTABLE_SAMPLE_RATE) ? "warning" : "success"}>
+                          Sampling transactions at {this.state.sentryPerformanceSampleRate}%
+                        </span>
+                      </span>
                     ) : (
-                      <p className="location warning">
+                      <span className="location warning">
                         Does not use Sentry performance!
-                      </p>
+                      </span>
                     )}
                   </li>
                   <li>
-                    <p className="text-muted location">
+                    <span className="text-muted location">
+                      Sending events to <b>{this.state.dsn.includes(SENTRY_DOMAIN) ? "SaaS" : "Self-Hosted Sentry"}</b> via <a href={this.state.dsn}>{this.state.dsn}</a>
```

Comment:
@cstavitsky 
It may still be going to sentry.io but via nginx reverse proxy / etc.

How about we just output the domain of the DSN and call it a day.
-  i.e. sending to sentry.io as project ID 123
- i.e. sending to nginx.reddit.com/sentry as project ID 123


@cstavitsky

Reply:
with the base domain, folks can then investigate further to see if its truly targeting on-premise or saas

Reply:
@ndmanvar Ahhh didn't think of that, very good point...

> i.e. sending to sentry.io as project ID 123
> i.e. sending to nginx.reddit.com/sentry as project ID 123

do we need to provide any instructions to AEs in the plugin itself (i.e. link to admin projects page)? Are AEs aware of how to look up project IDs in Admin and determine whether those are SaaS or Self-hosted?

Reply:
nope, don't worry about linking the AEs to the right info.

Some / most are not, but we can train them on it.
(also this is a public repo so we can't link them to our admin/etc)

Reply:
@cstavitsky